extContractsGroup.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "contivModel",
+	"objects": [
+		{
+			"name": "extContractsGroup",
+			"type": "object",
+			"key": [ "contractsGroupName" ],
```

Comment:
I think we want the key to be tenant, contractsGroupName. @jainvipin what do you think?